xds/internal/balancer/priority/balancer_priority.go

```diff
@@ -162,6 +162,9 @@ func (b *priorityBalancer) handleChildStateUpdate(childName string, s balancer.S
 		b.logger.Warningf("priority: child balancer not found for child %v", childName)
 		return
 	}
+	if !child.started {
+		return
```

Comment:
Maybe at least an info log about receiving an update from a child we stopped?  It's a race but could be of interest.

Reply:
Good point. I added it as a warning log because:
a. it is not a very common event, so won't flood the logs
b. if at all we are looking at a problem that has already happened and we have logs, there is more chance of this statement showing up as a warning log than as an info log.